medicines/doc-index-updater/src/service_bus_client/mod.rs

```diff
@@ -97,6 +97,8 @@ where
 pub enum ProcessMessageError {
     #[error(transparent)]
     SftpError(#[from] SftpError),
+    #[error("Cannot find document with ID {0}")]
+    DocumentNotFoundInIndex(String),
```

Comment:
@majikandy I was able to replace your custom error with just these two lines now that there's this `ProcessMessageError` in place